src/Core/Components/List/FluentAutocomplete.razor.cs

```diff
@@ -513,6 +513,34 @@ async Task KeyDown_EnterAsync()
         }
     }
 
+    /// <summary />
+    protected override void OnParametersSet()
+    {
+        base.OnParametersSet();
+
+        if (MaximumSelectedOptions == 1
+            && SelectedOptions?.Count() > 0
+            && !(AdditionalAttributes?.TryGetValue("single-select", out var isSingleSelect) ?? false))
+        {
+            var additionalAttributes = (AdditionalAttributes is not null)
```

Comment:
You don't need to update the `AdditionalAttributes`. You can add an attribute in the razor file like this:
```xml
<FluentTextField role="combobox" single-select="@GetSingleSelect()" ... >
```

```c#
private bool GetSingleSelect() => MaximumSelectedOptions == 1 && SelectedOptions?.Any() && Multiple == false;
```

Reply:
Done